Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [workspace]
 members = ["src/clippy-tracing", "src/cpu-template-helper", "src/firecracker", "src/jailer", "src/log-instrument", "src/log-instrument-macros", "src/rebase-snap", "src/seccompiler", "src/snapshot-editor"]
-default-members = ["src/firecracker"]
+default-members = ["src/clippy-tracing", "src/cpu-template-helper", "src/firecracker", "src/rebase-snap", "src/seccompiler", "src/snapshot-editor"]
```

Comment:
Can we not just remove `default-members`? It seems simplest to just get rid of this complexity. Do we lose any value removing it?

Reply:
Yeah, that was my first approach, but it seems that originally, the jailer was intentionally excluded from this (see the commit linked in my commit), so I decided to keep it like that while adding the binaries that we actually want to be able to execute.

Reply:
I think we should remove `default-members` or we should document why specific items are in `default-members`.